Automattic/wp-calypso
# Changing the displayed label status text based on image cache.

#### Changes proposed in this Pull Request

This PR builds upon https://github.com/Automattic/woocommerce-connect-server/pull/1228 in the context of https://github.com/Automattic/woocommerce-services/issues/1565 and https://github.com/Automattic/wp-calypso/pull/30086.

The only change is the usage of the `label_cached` parameter, which we receive from the server in order to use the appropriate label status string like this:

```js
const statusText = labelCached
	? translate( '%(service)s label (#%(labelIndex)d) printed', { args: statusArgs } )
	: translate( '%(service)s label (#%(labelIndex)d) not printed', { args: statusArgs } );
```

#### Testing instructions

* Start the process of creating a new shipping label.
* Close the tab/window before the completion of the process.
* Reopen the tab/window and make sure that the status includes`not printed` after everything is fully loaded.
* Open the popover menu and check whether the text says `Print` or `Reprint`. It should be just `Print`.
